README.md

```diff
@@ -40,24 +46,131 @@ If you intend to use the software constantly, you can install for the user or at
 Installation for the user can be carried out by:
 
 ```bash
-$ cp h1_client.bin ~/.local/bin/h1
+$ cp h1-linux ~/.local/bin/h1
 $ h1
 ```
 
 Installation at the system level can be carried out by:
 
 ```bash
-$ cp h1_client.bin /usr/local/bin/h1
+$ sudo cp h1-linux /usr/local/bin/h1
+$ h1
+```
+
+### Alpine environment
+
+You need to install *libstdc++* package using apk:
+
+```bash
+$ sudo apk add libstdc++
+```
+
+Download the *h1-alpine* file from [Releases] using wget:
+
+```bash
+$ wget https://github.com/hyperonecom/h1-cli/releases/latest/download/h1-alpine
+```
+
+Then run the downloaded binary file after granting executables rights:
+
+```bash
+$ chmod +x h1-alpine
+$ ./h1-alpine
+```
+
+The application should work correctly.
+
+If you intend to use the software constantly, you can install for the user or at the system level.
+
+Installation for the user can be carried out by:
+
+```bash
+$ mkdir -p ~/.local/bin/
+$ export PATH=${PATH}:~/.local/bin/ >> ~/.bash_profile
+$ source ~/.bash_profile
+$ cp ./h1-alpine ~/.local/bin/h1
+$ h1
+```
+
+Installation at the system level can be carried out by:
+
+```bash
+$ sudo cp h1-alpine /usr/local/bin/h1
 $ h1
 ```
 
 ### Windows environment
 
-> Introduction for Windows requires development taking into account the specificity of the platform.
+Download the *h1-win.exe* file from [Releases] using Invoke-WebRequest in Powershell:
+
+```bash
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Invoke-WebRequest https://github.com/hyperonecom/h1-cli/releases/latest/download/h1-win.exe -OutFile h1.exe
+```
+
+Then run the downloaded executable file using Powershell:
+
+```bash
+./h1
+```
+
+The application should work correctly without installing additional libraries / packages.
+
+If you intend to use the software constantly, you can add CLI to environment variables for the user or at the system level.
+
+Adding CLI to environment variables for the user can be carried out in Powershell by:
+
+```bash
+New-Item -ItemType directory -Path $ENV:UserProfile\h1\
+Copy-Item .\h1.exe $ENV:UserProfile\h1\
+[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "User") + ";$ENV:UserProfile\h1\", "User")
+h1
+```
+
+Adding CLI to environment variables at the system level can be carried out in Powershell by:
+
+```bash
+New-Item -ItemType directory -Path $ENV:Public\h1\
+Copy-Item .\h1.exe $ENV:Public\h1\
+[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "Machine") + ";$ENV:Public\h1\", "Machine")
```

Comment:
how different is this from using `$env:Path` ?

Reply:
$env:Path combines user path and system path